app/models/course.rb

```diff
@@ -516,6 +516,10 @@ def assignable_subjects
     services[:assignable_subjects].execute(course: self)
   end
 
+  def in_current_cycle?
+    recruitment_cycle == RecruitmentCycle.current
```

Comment:
This kicks off query to the db (maybe memoisation would help), but we also have a `current?` predicate.

```suggestion
    recruitment_cycle.current?
```

Reply:
`current?` does exist, I will use that instead